Snakefile

```diff
@@ -1,3 +1,7 @@
+from snakemake.remote.HTTP import RemoteProvider as HTTPRemoteProvider
+
+HTTP = HTTPRemoteProvider()
+
```

Comment:
This code is now extraneous after the switch to curl.

Reply:
Ah, I see this was removed in https://github.com/nextstrain/zika/commit/ebf528a4029fdb11c444c1a886390edc7967e980 already.